### DISCUSSION TITLE
Big bug in the version for mac os?

### DISCUSSION THREAD

**Discussion by luministdream** [Stellarium/stellarium#266] [Q&A] [answered]
Hello! Today I compared your stellarium program to mac os (ver. 01.18.2) and ios (ver. 1.23). The computer has a very large discrepancy with the data of the tablet. The same date and time, but different positions of constellations and sun at sunrise. For example: the date is 10,500 years BC 24 March. On the tablet like everything is exactly as it used to be. The sun rises after the rising of the constellation of the lion at the head of the Virgo constellation exactly in the east. Mercury and Venus follow the sun. On the computer, on the same date, the sun rises already in the constellation of twins along with the planet venus. To the right of the eastern side, closer to the southeast and about an hour later. Very strong discrepancy in the data. It seems to me that the error in the program on the computer.

**Comment by gzotti** [comment, marked as answer]
In 10.500BC both programs give errors of various size. I don't know who created the iOS version and cannot speak for it. The desktop version nowadays has accurate precession and could show accurate planets with the additional DE431 file installed. DeltaT can be expected to be uncertain by several hours.

What was special about March 24, -10499? My computer has Venus 45° ahead of the sun. On March 24, -10500 (which is March 24, 10501BC) Venus rises with the Sun,

How do you conclude which program is in error? What is your reference solution for this period? 
Start by comparison of Julian Day: -2113619 for the PC version.

**Comment by gzotti** [comment]
OP did not respond.
